### ISSUE TITLE
Allow changing model URL in Properties > Information tab

### ISSUE THREAD

**Issue by jbsarrodie** [archimatetool/archi-modelrepository-plugin#132]
In some occasion it might be needed to change the URL of the underlying git repository's origin. For the moment this has to be done by manually editing the git `config` file.

Would it be possible to make the `URL` field of the Property > Information tab editable ?

**Comment by Phillipus** [comment]
What would be the immediate consequences of doing this?

**Comment by Phillipus** [comment]
As an experiment, I tried doing this in SmartGit with two different repos to see what would happen if the user put in the URL of a completely different repo by mistake. I ended up with a combination of the two repos. So, not sure how this could be done safely.

**Comment by jbsarrodie** [comment]
>So, not sure how this could be done safely.

Maybe editing the `URL` text field is too quick and error prone, so some kind of confirmation dialog should appear to explain the risk (`Make sure that the new URL relates to the exact same model or else you're gonna die the next time you'll publish!`). If raising such dialog is not possible when simply editing the text field, then maybe we could add a menu entry to do this.

**Comment by jbsarrodie** [comment]
_FWIW, I had to do it twice with my team because we had to change our git server twice. I also have some colleague who has to work both from the LAN and a VPN, but depending on the network the port (thus URL) is not the same._

**Comment by Phillipus** [comment]
Yes, an edit button with a "Are you sure you know what you are doing?" warning might do it. If it's effectively only changing the remote entry in the `config` file there should be no consequences.

Have you tested what happens if one remote is ahead/behind the other, or has different branches?

**Comment by jbsarrodie** [comment]
>Have you tested what happens if one remote is ahead/behind the other, or has different branches?

Yes, there's no real issue in these case as this is anyway what happen if you did worked a lot locally while at the same time other people did create new branches. Git does its job ;-)
